src/figuresClasses.ts

```diff
@@ -1,11 +1,87 @@
-export interface Figure {}
+enum Shape {
+  'Triangle' = 'triangle',
+  'Circle' = 'circle',
+  'Rectangle' = 'rectangle',
```

Comment:
Enum constant names usually start with a capital letter. Consider renaming them to 'Triangle', 'Circle', and 'Rectangle' to follow this convention.